helm-chart/renku-gateway/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku gateway
 name: renku-gateway
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.1.0-a4fb826
+version: 0.0-unclean-cdf747b
```

Comment:
The chartpress relics shouldn't be committed - please revert to `version: 0.1.0` while you're at it...